### PR TITLE
Add className prop to badge component

### PR DESCRIPTION
**Pull request by psivakumaran**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "@dell/clarity-react",
-    "version": "0.22.1",
+    "version": "0.22.2",
```
Comment (akate1) [inline package.json]: @psivakumaran you can remove this. I have taken care of version changes in PR https://github.com/EMCECS/clarity-react/pull/93. I will create tar package for both changes once your PR gets merged.

File: src/forms/datalist/DataList.tsx [hunk 2]
```diff
@@ -82,6 +83,7 @@ type DataListProps = {
     helperText?: string;
     onChange?: (evt: React.ChangeEvent<HTMLInputElement>) => void;
     onBlur?: (evt: React.FocusEvent<HTMLInputElement>) => void;
+    autoComplete?: "on" | "off";
```
Comment (maheshkulkarni4) [inline src/forms/datalist/DataList.tsx]: Can we move these to enum and export that too.

File: src/forms/datalist/DataList.tsx [hunk 3]
```diff
@@ -177,6 +180,8 @@ export class DataList extends React.PureComponent<DataListProps, DataListState>
                                     ClassNames.NG_DIRTY,
                                 ])}
                                 aria-describedby="clr-form-control-3-helper"
+                                type="text"
+                                autoComplete={autoComplete ? autoComplete : "off"}
```
Comment (maheshkulkarni4) [inline src/forms/datalist/DataList.tsx]: instead of string off, use enum

File: src/forms/datalist/DataList.tsx (hunk 2 above)
Comment (psivakumaran) [reply src/forms/datalist/DataList.tsx]: Done

File: src/forms/datalist/DataList.tsx (hunk 3 above)
Comment (psivakumaran) [reply src/forms/datalist/DataList.tsx]: Done

**Pull request merged**